luigi/file.py

```diff
@@ -81,14 +81,16 @@ def __init__(self, path=None, format=None, is_tmp=False):
         self.format = format
         self.is_tmp = is_tmp
 
+    def mkdirs(self):
```

Comment:
Really short method name. Make it describe what you do. Like `create_parent_directory` or whatever you think suits.